projects/storefrontlib/shared/components/captcha/captcha.renderer.ts

```diff
@@ -0,0 +1,30 @@
+/*
+ * SPDX-FileCopyrightText: 2024 SAP Spartacus team <spartacus-team@sap.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { Observable } from 'rxjs';
+import { RenderParams } from './captcha.model';
+import { CaptchaConfig } from '@spartacus/core';
+
+export interface CaptchaRenderer {
+  /**
+   * Renders captcha widget
+   *
+   * @RenderParams parameters required by a provider, includes html element to append CAPTCHA UI element.
+   * @returns Observable for the key returned by a provider once user validated
```

Comment:
Why is it called *key* here? It's called *token* everywhere else. Or are these different things?